### DISCUSSION TITLE
variables in command

### DISCUSSION THREAD

**Discussion by ah-rahimi** [def-studio/telegraph#135] [General]
Hello 
I wanted to use variables in commands 
Like: /dl_42
The number 42 can change
And get that ID in the function in the code 
is there a way?

**Comment by fabio-ivona** [comment]
Don't have the code with me, but `\dl 42` should work, you listen for `dl` command and then read the parameter after the space

In future we may add support to parameters in a more structured way

**Reply by ah-rahimi** [reply]
no
i want use in message (user click on commands & send it)
how about inline auery 
i want show some meesge to user & if click it send a message with inline btn for download some file

**Reply by fabio-ivona** [reply]
What prevents you from sending `\dl 42`?

**Reply by ah-rahimi** [reply]
user search in bot then i sent a list 
i want use command on the list that user click on command send command 
if /dl 42 when user click on it just send /dl

**Reply by fabio-ivona** [reply]
Couldn't you use inline keyboard?

**Reply by ah-rahimi** [reply]
@fabio-ivona 
This way the page becomes very busy
But it becomes much easier with those types of commands
I have two types of commands (/pdf_42 and /zip_42) for each content that I display.

**Reply by fabio-ivona** [reply]
We will support command arguments, the feature will be implemented in next releases

**Reply by fabio-ivona** [reply]
added in next release
